prep-html-skeleton/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+  <html>
+
+    <title>HTML Skeleton Excersice</title>
+
```

Comment:
Hi Addy, your document is still missing a `<head>` section. That's where the `<title></title>` goes.

```html
<!DOCTYPE html>
<html>

  <head>
    <!-- meta info like <title></title> goes in here -->
  </head>

  <body>
    <!-- visible content like <h1></h1> and <p></p> goes in here -->
  </body>

</html>
```